docs/imagecustomizer/how-to/sysext.md

```diff
@@ -0,0 +1,145 @@
+
+Example mkosi.conf
+
+```yaml
+[Output]
+Format=sysext
+ImageId=kubernetes
+ImageVersion=1.30.7
+OutputDirectory=mkosi.output
+
+[Validation]
+Verity=signed
+VerityKey=verity_key.pem
+VerityCertificate=verity_cert.pem
+
+[Content]
+ExtraTrees=/usr/local/bin/kubectl:/usr/local/bin/kubectl, /usr/local/bin/kubelet:/usr/local/bin/kubelet
+```
+
+### [Output] Section
+- Format=sysext → Specifies that we are building a system extension image.
+- ImageId=kubernetes → Defines the image identifier, which is used in the output filename.
+- ImageVersion=1.30.7 → Sets the version number of the image, included in metadata and the default filename.
+- OutputDirectory=mkosi.output → Specifies the directory where the output file will be stored.
+
+### [Validation] Section
+The Verity= setting determines how your sysext image is secured:
+- signed → Fully signed image with both hash data and a cryptographic signature (requires VerityKey & VerityCertificate).
+- defer → Allocates space for a signature but does not populate it yet (useful for ESRP signing later).
+- hash → Hash verification only (no signature).
+- false → No verity at all (unsigned, unverified image).
+- VerityKey=verity_key.pem → The private key used for signing.
+- VerityCertificate=verity_cert.pem → The public certificate used for verification.
+
+### [Content] Section
+The ExtraTrees= setting specifies which binaries or files to include in the sysext image. \
+source_path:destination_path (comma separate for multiple entries) \
+/usr/local/bin/kubectl:/usr/local/bin/kubectl → Copies kubectl binary.\
+/usr/local/bin/kubelet:/usr/local/bin/kubelet → Copies kubelet binary.
+
+## Build and validate the Image
+```
+./bin/mkosi --force
+```
+
+When a sysext image is built with the configurations above, it contains three partitions:
+
+| Device             | Start Sector | End Sector | Size   | Type                          |
+|------------------------------|-------------|-----------|--------|-------------------------------|
+| kubernetes.raw1 | 2048        | 79983     | 38.1M  | Linux root (x86-64)           |
+| kubernetes.raw2     | 79984       | 100463    | 10M    | Linux root verity (x86-64)    |
+| kubernetes.raw3       | 100464      | 100495    | 16K    | Unknown                       |
+
+
+- kubernetes.raw1	main filesystem partition (contains kubectl and kubelet.)
+- kubernetes.raw2	contains the Merkle tree hash data used to verify the integrity of the main filesystem partition. 
+- kubernetes.raw3	contains the verity signature.
+
+# Integrate Sysext image into base image through Prism
+
+As part of its image customization process., Prism can include the .raw image into /var/lib/extensions/, along with public certificate (verity_cert.pem) used for signing the sysext.
+
+  ```yaml
+    # config.yaml
+    os:
```

Comment:
`os` should be a parent of `additionalFiles`. Currently, they have the same indent level.